ci-operator/step-registry/upi/install/powervs/sno/upi-install-powervs-sno-commands.sh

```diff
@@ -414,7 +415,7 @@ EOF
 
 cat > ${BASTION_CI_SCRIPTS_DIR}/grub-menu.template << EOF
     if [ \${GRUB_MAC_CONFIG} = "\${MAC_ADDRESS}" ]; then
-        linux \${KERNEL_PATH} ignition.firstboot ignition.platform.id=metal 'coreos.live.rootfs_url=\${ROOTFS_URL}' 'ignition.config.url=\${IGNITION_URL}'
+        linux \${KERNEL_PATH} ignition.firstboot ignition.platform.id=metal 'coreos.live.rootfs_url=http://192.168.140.2/\${ROOTFS_FILE}' 'ignition.config.url=http://192.168.140.2/\${IGNITION_FILE}'
```

Comment:
Should this be parameterized? It seems like the 192.168.140.2 IP might change over time

Reply:
The IP is for bastion's private IP, I'll make a env parameter for it.